Updates dated language on the main website (#37)
apache/httpd-site: content/index.md

Title: Welcome!
license: https://www.apache.org/licenses/LICENSE-2.0

# Apache HTTP Server

> A fast, reliable, and extensible open-source web server for modern
> operating systems. Apache httpd powers everything from personal sites to
> mission-critical production infrastructure with:

Reviewer: Is Novell NetWare a “modern operating system”?😁 I feel the portability point already addresses that, so I suggest just:
```suggestion
The Apache HTTP Server (httpd) is a fast, reliable, and extensible open-source
web server which powers everything from personal sites to
mission-critical production infrastructure with:
```

- **Flexible configuration** — .htaccess per-directory config, virtual hosts, dynamic module loading
- **Security** — TLS/SSL, authentication modules, fine-grained access control

> - **Performance** — event MPM, HTTP/2, content caching, reverse proxy

Reviewer: MPMs are an Apache httpd concept. Someone who does not know httpd won't even know what MPM means, let alone what the event MPM is, so:

```suggestion
- **Performance** — HTTP/2, content caching, reverse proxy
```

2.4](https://downloads.apache.org/httpd/CHANGES_2.4) | [New Features in httpd
2.4](docs/trunk/new_features_2_4.html)  {.centered}

# Apache httpd 2.2 End-of-Life

> Apache httpd 2.2 is end-of-life. The final release was 2.2.34 (July 2017).
> No further patches, bug fixes, or security updates will be provided.
> Users should upgrade to 2.4.x.

Reviewer: As far as I know, EOL is not an adjective.
```suggestion
Apache httpd 2.2 reached end-of-life. The final release was 2.2.34 (July 2017).
No further patches, bug fixes, or security updates will be provided.
Users should upgrade to 2.4.
```

# Download

> Get the latest release from the [download page](/download.cgi).
> Please [verify](/download.cgi#verify) your downloads.

Reviewer: I know this is entirely pre-existing, but if I may, the verification sentence is superfluous (and funny). The download page already asks for that in bold.
```suggestion
Get the latest release from the [download page](/download.cgi).
```
